src/bashly.yml

```diff
@@ -29,6 +29,12 @@ commands:
     help: Start date for analyze - only commits older than this date will be listed. Defaults to 01/01/2013.
     default: 01/01/2013
     required: false
+  - long: --end-date
+    short: -e
+    arg: date
+    help: End date for analyze - only commits younger than this date will be listed. Defaults to 31/12/2022.
+    default: 31/12/2022
```

Comment:
It means that if I do not provide this argument, I would receive results limited to some arbitrary date of `31/12/2022`? I think the expected result when no parameters are given would be that we receive all commits.

Reply:
If you don't provide `-d`, you also get an arbitrary start date very early. Would you be ok to provide arbitrary future dates e.g. 01/02/2420?

Reply:
Not perfect, but it would be consistent, and giving no parameters for start/end would result in all contributions (which is expected). Changed that way.